test/vtk/export_and_import_mesh.morpho

```diff
@@ -0,0 +1,16 @@
+import vtk 
```

Comment:
This test looks redundant with ensurevtkfilename.

Do you want to be checking m1 against m2?